vendor/k8s.io/kube-openapi/pkg/handler/handler.go

```diff
@@ -238,23 +238,6 @@ func jsonToYAMLValue(j interface{}) interface{} {
 			ret[i] = jsonToYAMLValue(j[i])
 		}
 		return ret
-	case float64:
```

Comment:
This is wrong, we need to update `glide.yaml` to pick up the correct (ours) version of `kube-openapi`.